status-im/status-react
# Fixes 3869: Add support for ATT for easy testing

Fixes #3869

### Summary:

Created a new contract on Ropsten that will simply send you 1000 `ATT` every time you send a 0 ETH tx to it, just to make Nastya's life easier and more token-filled when she is testing.
Added to Aragon logo and changed the address to the updated one - no other changes.

### Steps to test:
- Open Status
- Make sure you are on Ropsten
- Navigate to the wallet, check that you can see ATT.
- Send a 0 ETH tx to this address and check that your ATT balance increases by 1000: 0x00a8e52df8f4f1f4b67bded9ae6090b35489a973

status: ready
